eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -20,11 +20,6 @@ parameters:
   - name: Matrix
     type: object
     default:
-      Linux_Node8:
-        Pool: $(LinuxPool)
-        OSVmImage:
-        NodeTestVersion: "8.x"
-        TestType: "node"
```

Comment:
Node 8 doesn’t like how I’m using `URL`. We have alternatives, but since we dropped Node 8 support on v1.4.0 (2021-07-09), I think it’s safe to remove it from the test environments for Identity v1.

Reply:
This is fine by me. Additionally, the readme still talks about Node.js 8. Can you make the below changes to the readme?
- Remove the line `Node.js 8 LTS or higher.` under **Prerequesites**
- Add the line `See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.` under **Currently supported environments**

Reply:
I hope I did this right! https://github.com/Azure/azure-sdk-for-js/pull/16878/commits/3848c5b927ac7f4cea4a888354dd5a9440752155 👻

Reply:
Looks good, thanks!